internal/cli/ex-get-slots.go

```diff
@@ -0,0 +1,58 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/coreos/airlock/internal/lock"
```

Comment:
Drop this import to a block below the cobra import to follow what is done elsewhere in the repo.